### PR TITLE
Remove transfer cooldown from weapons

### PR DESCRIPTION
**Pull request by brian-man**

### CODE REVIEW

**Comment by brian-man** [comment]
It wasn't clear to me if it's as simple as just removing the plumbing, or if some of that plumbing has to be retained but neutered.  Please advise.

File: contracts/weapons.sol [hunk 1]
```diff
@@ -95,8 +89,6 @@ contract Weapons is Initializable, ERC721Upgradeable, AccessControlUpgradeable,
     int128 public powerMultPerPointPWR; // 0.2575% (+3%)
     int128 public powerMultPerPointMatching; // 0.2675% (+7%)
 
-    mapping(uint256 => uint256) public override lastTransferTimestamp;
```
Comment (raymond-h) [inline contracts/weapons.sol:98]: State variables cannot be removed, ever. The `override` keyword should be fine to remove however.

File: contracts/weapons.sol [hunk 2]
```diff
@@ -6,17 +6,13 @@ import "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol"
...
-import "./interfaces/ITransferCooldownable.sol";
 
-contract Weapons is Initializable, ERC721Upgradeable, AccessControlUpgradeable, ITransferCooldownable {
+contract Weapons is Initializable, ERC721Upgradeable, AccessControlUpgradeable {
 
     using ABDKMath64x64 for int128;
     using ABDKMath64x64 for uint16;
 
     bytes32 public constant GAME_ADMIN = keccak256("GAME_ADMIN");
-    bytes32 public constant RECEIVE_DOES_NOT_SET_TRANSFER_TIMESTAMP = keccak256("RECEIVE_DOES_NOT_SET_TRANSFER_TIMESTAMP");
```
Comment (raymond-h) [inline contracts/weapons.sol:17]: I would like to keep this constant for now, as it is used in `migrations/29_transfer_cooldown.js`. While that is an old migration, and this constant won't actually be applicable in any way after this PR is merged, I would like to leave all applied migration files untouched for now.

Eventually I would like to merge all the old applied migrations we have right now into a single file - I think that would be a good time to finally remove this constant.

**Review by raymond-h — CHANGES_REQUESTED** [review]
Largely good, but the frontend part of the weapon transfer cooldown code still has to be removed as well.

File: contracts/weapons.sol (hunk 1 above)
Comment (brian-man) [reply contracts/weapons.sol:98]: Okies.
Is there some language sugar or comment you recommend to make it clear it's no longer to be used?

Comment (raymond-h) [reply contracts/weapons.sol:98]: I guess use `// UNUSED; KEPT FOR UPGRADEABILITY PROXY COMPATIBILITY`

**Review by raymond-h — CHANGES_REQUESTED** [review]
Looks good to me now, just needs frontend work now.

**Review by raymond-h — APPROVED** [review]

**Pull request merged**